zkevm-circuits/src/evm_circuit/util.rs

```diff
@@ -469,6 +469,14 @@ impl<'a> StepRws<'a> {
     pub(crate) fn offset_add(&mut self, inc: usize) {
         self.offset += inc
     }
+    /// Decrement the step rw operation offset by `dec`.
+    pub(crate) fn offset_sub(&mut self, dec: usize) {
+        self.offset -= dec
+    }
+    /// Reset the step rw operation offset to 0.
+    pub(crate) fn offset_reset(&mut self) {
+        self.offset = 0
+    }
```

Comment:
These two APIs are no longer consumed in the codebase.

We shouldn't have API to decrease or rest offset for the reasons discussed in my previous comments.
```suggestion
```